src/contracts/powerSwitch/PowerSwitch.sol

```diff
@@ -46,6 +46,8 @@ interface IPowerSwitch {
 }
 
 /// @title PowerSwitch
+/// @notice This power switch contract receives a uint256 
+///         to enforce powering a switch only after a particular timestamp.
```

Comment:
if there's any difference with other powerswitchs I feel we should document that